packages/xchain-bitcoin/src/client.ts

```diff
@@ -148,13 +140,15 @@ class Client implements BitcoinClient, XChainClient {
   }
 
   /**
-   * Get DerivePath
+   * Get getFullDerivationPath
    *
+   * @param {number} index the HD wallet index
    * @returns {string} The bitcoin derivation path based on the network.
    */
-  derivePath(): string {
-    const { testnet, mainnet } = Utils.getDerivePath()
-    return Utils.isTestnet(this.net) ? testnet : mainnet
+  getFullDerivationPath(index: number): string {
```

Comment:
Make `index` optional and set it to `0` by default. Just to keep it similar with other methods, which expect `index`.

Reply:
i feel like gertign the derivation path should NOT default to zero, so goign to leave this one